src/pages/GrantProgram/SingleProgram/LeaderBoardSection/index.tsx

```diff
@@ -26,6 +25,18 @@ const EmptyRankings: any[] = []
 
 export const ITEMS_PER_PAGE = 5
 
+const formatTradingVolume = (num: number) => {
```

Comment:
same as `formatDollarAmount` or not?

Reply:
It's a bit different. Only use compact when it > 100M and use 0 fraction digits